public/static-checkout/js/scripts.js

```diff
@@ -108,8 +111,8 @@ window.onclick = function closeMenu(event) {
 };
 
 function redirectToMFE(couponCode) {
-  var url = window.location.href;
-  var sku = /static-checkout\/([^&#]*).html/.exec(url)[1];
+  var url = window.location.pathname;
+  var sku = /.*\/([^&#]*).html/.exec(url)[1];
```

Comment:
this regex wasn't working for me

Reply:
Why wasn't it working? I'm curious because I get the sku when I try on console

Reply:
not sure, i get that now too but pretty sure at one point it didn't work